modules/vertx-swagger-codegen/src/main/resources/javaVertXServer/apiRXVerticle.mustache

```diff
@@ -0,0 +1,85 @@
+import java.util.Map;
+
+public class {{classname}}Verticle extends AbstractVerticle {
+    final static Logger LOGGER = LoggerFactory.getLogger({{classname}}Verticle.class); 
+    
+    {{#operations}}{{#operation}}{{#vendorExtensions}}final static String {{x-serviceId-varName}} = "{{x-serviceId}}";
+    {{/vendorExtensions}}{{/operation}}{{/operations}}
+    //TODO : create Implementation
+    {{classname}} service = new {{classname}}Impl();
+
+    @Override
+    public void start() throws Exception {
+        {{#operations}}{{#operation}}
+        //Consumer for {{#vendorExtensions}}{{x-serviceId}}{{/vendorExtensions}}
+        vertx.eventBus().<JsonObject> consumer({{#vendorExtensions}}{{x-serviceId-varName}}{{/vendorExtensions}}).handler(message -> {
+            try {
+                {{#hasParams}}
+                    {{#allParams}}
+                        {{#isListContainer}}
+                {{{dataType}}} {{paramName}} = Json.mapper.readValue(message.body().getJsonArray("{{baseName}}").encode(), Json.mapper.getTypeFactory().constructCollectionType(List.class, {{{baseType}}}.class));
+                        {{/isListContainer}}
+                        {{^isListContainer}}
+                            {{#isPrimitiveType}}
+                                {{#isString}}
+                {{{dataType}}} {{paramName}} = message.body().getString("{{baseName}}");
+                                {{/isString}}
+                                {{^isString}}
+                {{{dataType}}} {{paramName}} = Json.mapper.readValue(message.body().getString("{{baseName}}"), {{{dataType}}}.class);
+                                {{/isString}}
+                            {{/isPrimitiveType}}
+                            {{^isPrimitiveType}}
+                {{{dataType}}} {{paramName}} = Json.mapper.readValue(message.body().getJsonObject("{{baseName}}").encode(), {{{dataType}}}.class);
+                            {{/isPrimitiveType}}
+                        {{/isListContainer}}
+                    {{/allParams}}
+                {{/hasParams}}
+                service.{{operationId}}({{#hasParams}}{{#allParams}}{{paramName}}{{#hasMore}}, {{/hasMore}}{{/allParams}}{{/hasParams}}).subscribe(
+                    onSuccess -> {
+                {{#returnType}}
+                    {{#isListContainer}}
+                        message.reply(new JsonArray(Json.encode(onSuccess)).encodePrettily());
+                    {{/isListContainer}}
+                    {{^isListContainer}}
+                        message.reply(new JsonObject(Json.encode(onSuccess)).encodePrettily());
+                    {{/isListContainer}}
+                {{/returnType}}
+                {{^returnType}}
+                        message.reply(null);
+                {{/returnType}}
+                    },
+                    onError -> {
+                        LOGGER.error("Error in " + {{#vendorExtensions}}{{x-serviceId-varName}}{{/vendorExtensions}}, onError);
```

Comment:
I'd say we define our error handling approach and then follow it in all mustache files.

My current recommendation is to log all unexpected errors, so errors that are not of type MainApiException. I did it like this in #34. Let me know, what you think.